tutorials/sinon/check-arguments.md

```diff
@@ -0,0 +1,95 @@
+To check what arguments a stub was called with, you have several options as stubs are treated as spies in sinon.js.
+
+
+## calledWith()
+
+`calledWith` will return true if the stub was called at least once with the provided arguments.
+The function can also return true if the stubbed function had more arguments than it should have, but are correctly positioned.
+
+```javascript
+const sinon = require('sinon');
+
+module.exports = {
+    add: (a,b) => {
+        return a+b;
+    }
+};
+async function run() {
+const stub = sinon.stub(module.exports, 'add');
+module.exports.add(1,2);
+stub.calledWith(1,2); // true
+stub.calledWith(0,5); // false
+module.exports.add(1,2,3,4,5); // 3 arguments too many
+stub.calledWith(1,2); // true, because 1 and 2 are in the same position as the stub call
+}
+
+run();
+```
+
+## calledOnceWith()
```

Comment:
Combine `calledOnceWith()` and `alwaysCalledWith()` examples and move the consolidated example into the `calledWith()` section. I'd prefer to keep them as part of `calledWith()` since they're just helpers on top of `calledWith()`, whereas `getCall()` is different.